articles/azure-arc/servers/overview.md

```diff
@@ -6,7 +6,7 @@ ms.date: 02/18/2021
 ms.topic: overview
 ---
 
-# What is Azure Arc enabled servers?
+# What are Azure Arc enabled servers?
```

Comment:
This change isn't using proper grammar given the title is referring to the formal/proper name of the service feature - **Arc enabled servers**. Please change "are" back to "is". Thanks.